azure-pipelines.yml

```diff
@@ -0,0 +1,92 @@
+parameters:
+  # Test only the Release build by default.
+- name: ReleaseBuildTarget
+  displayName: 'Build which target for Release?'
+  type: string
+  values: [ Build, Integration, UnitTest ]
+  default: UnitTest
+- name: OtherBuildTarget
+  displayName: 'Build which target for Debug/CodeAnalysis?'
+  type: string
+  values: [ Build, Integration, UnitTest ]
+  default: Build
+
+variables:
+- name: DOTNET_CLI_TELEMETRY_OPTOUT
+  value: 1
+  # Run CodeQL3000 tasks in a separate internal pipeline; not needed here.
+- name: Codeql.SkipTaskAutoInjection
+  value: true
+
+trigger: [main]
+pr: ['*']
+
+jobs:
+- job: build
+  displayName: Build
+  pool:
+    ${{ if eq(variables['System.TeamProject'], 'public') }}:
```

Comment:
So is this what differentiates whether the build was kicked-off from AzDO vs TFS ?

Reply:
Ok, based on the later comment I see that this is used to differentiate between internal and public builds